src/ui/osx/TogglDesktop/Feature/TimeEntry/Views/TimeEntryCollectionView.m

```diff
@@ -32,33 +33,30 @@ - (BOOL)isFlipped
 
 - (void)mouseUp:(NSEvent *)event {
```

Comment:
This method doesn't execute in macOS 10.11 when the user selects the cell by mouse. It results in the first issue in this ticket, which is "Unable to select cell in Time Entry List"

Therefore, I moved this "Selection logic" to 
```swift
func collectionView(_ collectionView: NSCollectionView, didSelectItemsAt indexPaths: Set<IndexPath>)
```
So it works.

However, I'm struggling when I couldn't re-select the cell again on macOS 10.11 because this logic
https://github.com/toggl/toggldesktop/blob/b95cdd9fff681f2406dafad1a7511801e3ce5a09/src/ui/osx/TogglDesktop/Feature/TimeEntry/Views/TimeEntryCollectionView.m#L43
 doesn't execute